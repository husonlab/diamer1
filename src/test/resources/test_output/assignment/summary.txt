Total reads: 6

OVO (ratio: 0.200000)
Total assignments:	3 (0.5%)
1 from root: 1
no rank: 1
2 from root: 1

OVO (ratio: 0.800000)
Total assignments:	3 (0.5%)
1 from root: 1
no rank: 1
2 from root: 1

