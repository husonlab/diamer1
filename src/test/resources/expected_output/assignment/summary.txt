Total reads: 6

OVO (ratio: 0.200000)
Total assignments:	4 (0.7%)
1 from root: 2
no rank: 1
2 from root: 1

OVO (ratio: 0.800000)
Total assignments:	4 (0.7%)
1 from root: 2
no rank: 1
2 from root: 1

